cssdb.settings.json

```diff
@@ -894,12 +1007,16 @@
     "description": "A property for defining whether to insert line breaks within words to prevent overflowing",
     "specification": "https://www.w3.org/TR/css-text-3/#overflow-wrap-property",
     "stage": 2,
-    "caniuse": "wordwrap",
     "browser_support": {},
     "docs": {
       "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap"
     },
     "example": "p {\n  overflow-wrap: break-word;\n}",
+    "mdn_path": [
+      "css.properties['overflow-wrap']",
+      "css.properties['overflow-wrap'].anywhere",
```

Comment:
Is `anywhere` part of the plugin?

Reply:
No, and this is kind of recurring. The plugin might be incomplete and I thought to be greedier with the spec rather than plugin-specific given that we don't know when the deviation happened or if it's just that the spec got bigger.

Reply:
Sound good!

In practice it would mean that we apply the plugin and transform in `postcss-preset-env` maybe a little bit more than needed, but never the other way around?

Reply:
Exactly! My reasoning was that often specs are implemented in chunks and it'd be rare for subfeatures not to be available. Given we're also more aware now, we can reassess though at any given time